.github/workflows/auto_approve_dependency_PRs.yml

```diff
@@ -0,0 +1,37 @@
+name: Auto Approve Dependency PRs
+on:
+  schedule:
+    - cron: '*/30 * * * *'
+  workflow_dispatch:
```

Comment:
What does this do?

Reply:
This means you can go the actions tab and manually kickoff this workflow:

<img width="1253" alt="Screen Shot 2022-01-05 at 7 10 56 PM" src="https://user-images.githubusercontent.com/8726321/148307693-1611d447-1f65-4992-a44f-618f38d8e8e8.png">